aries-site/src/pages/design-tokens/layout-and-spacing.mdx

```diff
@@ -0,0 +1,93 @@
+
+This results in the following: 1, 2, 3, 4, 6, 8, 12, 16, 24, 32, 48, 64, 96, 128, 192, 256, 384, 512, 768, 1024, 1536.
+
+Spacing, container, radius, and borderWidth tokens are created using a subset of this scale.
+
+## Spacing
+
+Spacing uses proximity principles to create a rhythm and hierarchy within a page. Spacing design tokens define the padding within, margin around, or gap between content.
+
+Rather than relying heavily on borders to separate content, spacing creates a visual hierarchy that guides the user’s eye through the page and helps them understand the relationships between different elements.
+
+There are 12 spacing tokens to provide flexibility to cater to various design needs ranging from dense data-rich experiences to more spacious layouts.
+
+<ContentPane>
+  <DesignTokensTable active="semantic.spacing" toolbar />
+</ContentPane>
+
+### Spacing best practices
+
+<BestPracticeGroup>
+  <Example
+    height="none"
+    bestPractice={{
+      type: 'do',
+      message:
+        'Use spacing design tokens to style compositional or layout components like cards, layers, and grid spacing.',
+    }}
+    width="100%"
+  >
+    <SpacingBestPractices />
+  </Example>
+  <Example
+    height="none"
+    bestPractice={{
+      type: 'dont',
+      message: `Use spacing design tokens to style element-level components. Instead use component-specific tokens or element tokens which will ensure correct sizing of elements across screen sizes.`,
+    }}
+  >
+    <SpacingBestPractices bestPractice={false} />
+  </Example>
+</BestPracticeGroup>
+
+## Containers
+
+Containers define the minimum/maximum width or height of a page section.
+
+Container tokens are particularly useful when defining grid layouts. In other cases, consider the approach of [content-driven layouts](/templates/content-layouts#content-driven-layouts) to avoid artificially constraining content.
+
+There are TBD container tokens to provide flexibility to cater to various design needs.
+
+<ContentPane>
+  <DesignTokensTable active="semantic.container" />
+</ContentPane>
+
+## Radius
+
+Radius tokens define the rounding of corners on elements. Radius can be used as a tool to further emphasize the shape or relationship between certain elements.
+
+Larger elements, like page sections, may have a larger radius while elements nested within may have smaller radii.
+
+<ContentPane>
+  <DesignTokensTable active="semantic.radius" toolbar />
+</ContentPane>
+
+## Responsiveness
+
+Layouts should be designed with responsiveness in mind to meet a user wherever they are— whether on a desktop, tablet, or mobile device.
+
+The values of spacing and radius tokens automatically step down one t-shirt size at the `small` [breakpoint](/design-tokens/global#breakpoint). This retains appropriate proportions between shape, spacing, and content as the screen size decreases.
```

Comment:
so when the breakpoint is small then the radius and tokens would be one smaller than small? Im not sure I understand this part

Reply:
Let me clarify the wording here.

Reply:
Done.